CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -110,7 +110,7 @@ CF_EXTERN_C_BEGIN
 #include <pthread.h>
 
 #if !DEPLOYMENT_RUNTIME_SWIFT && __has_include(<os/log.h>)
-#import <os/log.h>
+#include <os/log.h>
```

Comment:
Does windows have <os/log.h>?

Reply:
No, it doesn't, just going through an getting them all in a single go.